elements/lisk-cryptography/test/nacl/index.ts

```diff
@@ -109,25 +109,11 @@ describe('nacl index.js', () => {
 			return expect(process.env.NACL_FAST).to.eql('disable');
 		});
 
-		it('should load nacl slow if process.env.NACL_FAST is set to enable', () => {
```

Comment:
this test seems to be wrong, but why it was passing?

Reply:
The Test check the interface of the loaded module which is equal for slow and fast implementation.
```
{
  box: [Function],
  openBox: [Function],
  signDetached: [Function],
  verifyDetached: [Function],
  getRandomBytes: [Function],
  getKeyPair: [Function],
  getPublicKey: [Function]
}
```

Reply:
Related issue about tests #4677